samples/openapi3/client/petstore/python/tests/test_rest.py

```diff
@@ -0,0 +1,31 @@
+import json
+import unittest
+from unittest.mock import patch, Mock
+
+import pytest
+
+import petstore_api
+
+
+class TestMultipleResponseTypes(unittest.TestCase):
+    def setUp(self):
+        self.api_client = petstore_api.ApiClient()
+        self.fake_api = petstore_api.FakeApi(self.api_client)
+
+    def test_multipart_requests(self):
+        mock_resp = Mock()
+        mock_resp.status = 200
+        mock_resp.data = b"some text"
+        mock_resp.headers = {}
+
+        marker = petstore_api.TestObjectForMultipartRequestsRequestMarker(
+            name="name",
+        )
+
+        with patch(
+            "urllib3.PoolManager.urlopen", return_value=mock_resp
```

Comment:
`urllib3.PoolManager.urlopen` can be mocked to test the arguments to `urllib3.PoolManager.request` without making an http request

Reply:
what about not using Mock but instead use the echo API client (python) for testing as the echo server can return whatever sent by the client?

Reply:
I initially considered using the echo API, but gave up because the Python client inputSpec is not independent!
When I add a test, it also generates sample code for other unrelated languages.

https://github.com/OpenAPITools/openapi-generator/blob/e2df0d615409e07256486ca631cb27b8e5906f57/bin/configs/python-echo-api.yaml#L3

Should I add it as is or make it independent?

Also, I was able to test it using Mock, so I might as well leave it as it is.

Reply:
please go ahead to add it to echo_api.yaml and generate new files for other clients as well (which shouldn't break other clients as all)

in case your new tests uncover issues in other clients, we can then consider using a dedicated test spec for python for the time being.

Reply:
saw you added the test in echo_api and that pr has been merged.

let's include this one as well to have one more additional test.